docs/content/how-to/azure/create-self-managed-azure-cluster.md

```diff
@@ -47,7 +47,7 @@ Your Azure service principal must have the following permissions:
 Before creating the HostedCluster, set up the necessary Azure infrastructure:
 
 !!! note "About PERSISTENT_RG_NAME"
```

Comment:
Same as above, I think this is not the target-audience.

Reply:
I think this is fine to leave it here. We have Red Hat specific things in our upstream docs, this does not mean it will be included in the official docs.